.travis.yml

```diff
@@ -27,19 +27,22 @@ matrix:
       python: '3.7'
       install:
         - pip install -U pip setuptools "mypy~=0.740"
+        - pip install -U importlib-metadata~=5.0 # this needs to be installed separately BEFORE trying to parse the setup.py file
+        - pip install ".[mypy]"
```

Comment:
This line is here to make sure that mypy dependencies are explicitly installed in the environment before trying to run mypy in the 'script' section below. The test dependencies are kind of 'implicitly' installed whenever the `python setup.py test` command runs but dependencies installed that way don't seem to stick around in the python environment for use in future commands.